tests/end_to_end/test_import.py

```diff
@@ -188,7 +188,7 @@ def test_nested_from_import_starred(execute):
     code = """from matplotlib.pyplot import *
 x = ylabel('label')
 """
-    res = execute(code, artifacts=["x"])
+    res = execute(code, artifacts=["x"], snapshot=False)
```

Comment:
we're not using snapshot for assertions in this test (also snapshot for * does not make sense). edited this file to trigger runs. not removing since its a positive change.